README.md

```diff
@@ -3,13 +3,12 @@
 This project is in the initial availability phase.
 It is suitable for production usage.
 
-**Important:** Observability Logs (Beta) is enabled only for selected customers. If you’d like to participate, contact your Observability account representative.
+**Important:** Logs (Beta) is enabled only for selected customers. If you’d like to participate, contact your Observability account representative.
 
-# Overview of Wavefront Operator for Kubernetes
+# Overview of the Observability for Kubernetes Operator
 
-The Wavefront Operator for Kubernetes
-supports deploying the Wavefront Collector and the Wavefront Proxy in Kubernetes.
-This operator is based on [kubebuilder SDK](https://book.kubebuilder.io/).
+The Observability for Kubernetes Operator supports deploying the Kubernetes Metrics Collector and the Wavefront proxy in Kubernetes.
```

Comment:
I realize this wasn't part of what you're doing, but reading this, it is wrong. It should say something like "deploys the necessary agents to monitor your clusters and workloads in Kubernetes." a) it doesn't list logging, b) hopefully they don't care what the agents are, and c) why update it again when we add autoinstrumentation?